src/prefect/cli/work_pool.py

```diff
@@ -33,6 +34,14 @@
 @work_pool_app.command()
 async def create(
     name: str = typer.Argument(..., help="The name of the work pool."),
+    base_job_template: str = typer.Option(
+        None,
+        "--base-job-template",
+        help=(
+            "The base job template to use. If unspecified, Prefect will use the default"
+            " base job template."
+        ),
+    ),
```

Comment:
We should probably call out in the help text that the `--base-job-template` option is expecting a path to a `.json` file. Also, `typer` has a handy [`FileText`](https://typer.tiangolo.com/tutorial/parameter-types/file/#filetext-reading) type that can handling the file reading for you.

Reply:
Nice! Updated the docstring and switched to FileText https://github.com/PrefectHQ/prefect/pull/10793/commits/4ebee37902b26cc23626e36ff75721540e326c4e